src/components/Collection/scss/index.scss

```diff
@@ -0,0 +1,36 @@
+.data-table {
+  display: flex;
+  flex-direction: row;
+
+  &-info {
+    flex: 1 0 50%;
+    max-width: 50%;
+
+    &-box {
+      margin-bottom: 8px;
+      height: calc(100% - 8px);
+      border-top: 1px solid #e9e9e9;
+      border-bottom: 1px solid #e9e9e9;
+      align-items: flex-start;
+      display: flex;
+      justify-content: flex-start;
+      padding: 14px 12px;
```

Comment:
For units like margin, padding, you should use the $spacer variable instead or use rem units to keep things consistent.